packages/ui/src/lib/components/Notebook/NotebookCover.svelte

```diff
@@ -143,11 +143,6 @@
   style:--color-text-fallback={colorValue[2][1]}
   class:canClick={onclick !== undefined}
   {onclick}
-  onauxclick={(e) => {
-    if (onclick) {
-      onclick(e)
-    }
-  }}
```

Comment:
we need this to [handle middle mouse clicks](https://github.com/deta/horizon/pull/2276) to open the notebook in the background

maybe we need to ignore this event in certain cases?